source/js/components/nav/nav.jsx

```diff
@@ -26,51 +27,97 @@ export default class PrimaryNav extends React.Component {
+
+  componentDidMount() {
+    // Close primary nav when escape is pressed
+    document.addEventListener(`keyup`, this.escKeyPressed);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener(`keyup`, this.escKeyPressed);
+  }
+
+  renderBurger() {
+    let className = classNames({
+      'hidden-md-up': this.props.navMode !== "zen"
+    });
+
+    return (
+      <button id="burger" onClick={this.toggleMenu} className={className}></button>
+    );
+  }
+
+  renderLinks() {
+    let hidden = false;
+    if (this.props.navMode === "zen") {
+      hidden = !this.state.menuOpen;
+    }
+
+    return (
+      <div className="nav-links hidden-sm-down" hidden={hidden}>
+        <a className="nav-link-initiatives" href="/initiatives">Initiatives</a>
+        <a className="nav-link-participate" href="/participate">Participate</a>
+        <a href="https://internethealthreport.org" target="_blank" rel="noopener noreferrer">Internet&nbsp;Health</a>
+        <a className="nav-link-people" href="/people">People</a>
+        <a className="nav-link-about" href="/about">About&nbsp;Us</a>
+      </div>
+    );
+  }
+
   render() {
     return (
-      <header hidden={this.state.isHidden}>
+      <div className="wrapper-burger">
         <div className="container">
-          <div className="row p-4 d-flex justify-content-end">
-            <button className="btn-close text-hide" onClick={this.hide}>X</button>
-          </div>
-          <div className="row d-md-flex justify-content-between">
-            <div className="col-sm-auto align-self-end">
-              <ul className="px-5 menu-nav">
-                <li><a className="home" href="/">Home</a></li>
-                <li><a className="initiatives" href="/initiatives">Initiatives</a></li>
-                <li><a className="participate" href="/participate">Participate</a></li>
-                <li><a href="https://internethealthreport.org" target="_blank" rel="noopener noreferrer">Internet&nbsp;Health</a></li>
-                <li><a className="people" href="/people">People</a></li>
-                <li><a className="about" href="/about">About&nbsp;Us</a></li>
-              </ul>
+          <div className="row">
+            <div className="col py-4">
+              <div className="d-flex flex-row justify-content-between align-items-center">
+                  <div className="d-flex align-items-center flex-wrap">
+                    {this.renderBurger()}
+                    <a className="logo text-hide" href="/">
+                      Mozilla Network
+                    </a>
+                    {this.renderLinks()}
+                  </div>
+                  <div className="d-flex align-items-center">
+                    <a id="donate-header-btn"
+                      className="btn btn-pop"
+                      href="https://donate.mozilla.org?utm_source=foundation.mozilla.org&utm_medium=referral&utm_content=header"
+                      target="_blank" rel="noopener noreferrer"
```

Comment:
are we sure this should be _blank?

Reply:
Probably, I dunno. That's how it was before, and seems to make sense to me. Open to other values though, what did you have in mind?